docs/content/get-started/generate-providers.md

```diff
@@ -1,197 +0,0 @@
-   +                               Description: `UPDATED_DESCRIPTION`,
-   diff --git a/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown b/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown
-   --- a/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown
-   +++ b/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown
-   @@ -63 +63 @@ The following arguments are supported:
-   -  Human-readable display name of the data exchange. The display name must contain only Unicode letters, numbers (0-9), underscores (_), dashes (-), spaces ( ), and must not start or end with spaces.
-   +  UPDATED_DESCRIPTION
-   ```
-
-   {{< hint info >}}
-   **Note**: There may be additional changes present due to specifying a `PRODUCT=` value or due to the `magic-modules` repository being out of sync with the provider repositories. This is okay as long as tests in the following section pass.
-   {{< /hint >}}
-
-
-## Test changes
-
-1. Set up application default credentials for Terraform
-   ```bash
-   gcloud auth application-default login
-   export GOOGLE_USE_DEFAULT_CREDENTIALS=true
-   ```
-1. Set required environment variables
-   ```bash
-   export GOOGLE_PROJECT=PROJECT_ID
-   export GOOGLE_REGION=us-central1
-   export GOOGLE_ZONE=us-central1-a
-   ```
-   Replace `PROJECT_ID` with the ID of your Google Cloud project.
-
-1. Enable required APIs
-   ```bash
-   gcloud config set project $GOOGLE_PROJECT
-   gcloud services enable analyticshub.googleapis.com
-   ```
-1. Run all linters
-   ```bash
-   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
-   make lint
-   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta
-   make lint
-   ```
-1. Run all unit tests
-   ```bash
-   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
-   make test
-   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta
-   make test
-   ```
-1. Run acceptance tests for BigqueryAnalyticsHub DataExchange
-
-   ```bash
-   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
-   make testacc TEST=./google/services/bigqueryanalyticshub TESTARGS='-run=TestAccBigqueryAnalyticsHubDataExchange_'
-   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta
-   make testacc TEST=./google-beta/services/bigqueryanalyticshub TESTARGS='-run=TestAccBigqueryAnalyticsHubDataExchange_'
-   ```
-
-## Troubleshoot
-
-### Too many open files {#too-many-open-files}
-
-If you are getting “Too many open files” ulimit needs to be raised.
-
-{{< tabs "ulimit" >}}
-{{< tab "Mac OS" >}}
-```bash
-ulimit -n 8192
-```
-{{< /tab >}}
-{{< /tabs >}}
-
-## Cleanup
-
-1. Optional: Revoke credentials from the gcloud CLI.
-
-```bash
-gcloud auth revoke
-```
```

Comment:
it looks like this was lost - I don't know if we need it, but it could maybe go on the `run-tests` page

Reply:
Cleanup is usually recommended for quickstarts/tutorials where you "hand hold" the reader by giving them specific values to enter for fields/variables. IMO, it made sense to have this section when this page was written in the quickstart format, but I think its ok for us to delete it now. LMK.

Reply:
SGTM